tests/integration/misc-tosca-types/runme.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
We should be more strict when running this command. I would suggest that we change this to

    #!/bin/bash
    set -euo pipefail

This makes sure that the script will terminate after the first failed command. Because doing undeploy when messed-up the code for deploy makes little sense.

Reply:
Very good point, I will use the suggested changes.